src/index.js

```diff
@@ -27,144 +34,178 @@ const providers = {
   },
 };
 
-// Set up program details for -h and -v options
-program
-  .name(name)
-  .description(description)
-  .version(`${name} v${version}`, "-v, --version")
-  .option("-o, --output <filename>", "specify filename to write output to")
-  .option("-t, --token-usage", "report token usage")
-  .argument("<output-language>", "language to transform code to")
-  .argument("<input-files...>", "source files to read")
-  .action(async (outputLang, inputFiles) => {
-    if (!process.env.API_KEY) {
-      console.error(`missing expected env var: "API_KEY"`);
-      process.exit(20);
-    }
+// default values
```

Comment:
- Capitalize first letter for consistency
- Make more descriptive (Default values for what?)

Reply:
Done.